crates/agglayer-signer/src/tests.rs

```diff
@@ -0,0 +1,43 @@
+use ethers::types::{Eip1559TransactionRequest, H160};
+
+use super::*;
+
+async fn check_signer_works(signer: &ConfiguredSigner) {
```

Comment:
Why put this in a dedicated function and not directly in the test function?

Reply:
The idea is to have separate tests for different kinds of signers (wallet, gcp kms, etc) but reuse the test logic.

Reply:
Okey. at some point we could rely on rs-test fixture then

Reply:
> Okey. at some point we could rely on rs-test fixture then

I ported the test to rstest to prepare for this.